priv/priv.go

```diff
@@ -36,7 +36,8 @@ var (
 
 // Mutex is the snappy mutual exclusion primitive.
 type Mutex struct {
-	lock *FileLock
+	filename string
+	flock    LockedFile
```

Comment:
priv.Mutex is a misnomer, I think, and needs to die. In a later branch (as below), ideally.